torch/_decomp/__init__.py

```diff
@@ -255,6 +255,8 @@ def core_aten_decompositions() -> Dict[OpOverload, Callable]:
             aten._log_softmax_backward_data,
             aten.logspace,
             aten.logsumexp.default,
+            aten.__lshift__.Scalar,
+            aten.__lshift__.Tensor,
```

Comment:
You can probably just write
```suggestion
            aten.__lshift__,
```
to select them all.